understanding/20/keyboard.html

```diff
@@ -75,19 +75,18 @@ <h2>Intent of Keyboard</h2>
             the platform/user agent conventions it may be difficult to use, as users will need
             to learn different interaction methods. As a <em>best practice</em>, content
             should follow the platform/user agent conventions. However, deviating from these
-            conventions does <em>not</em> fail the normative requirement of this Success Criterion.</p>
+            conventions does <em>not</em> always fail the normative requirement of this Success Criterion.</p>
```

Comment:
Partially dealing with @patrickhlauke's feedback...

```suggestion
            conventions does <em>not</em> automatically fail the normative requirement of this Success Criterion.</p>
```

(The rest of his comment is addressed, IMHO, by the text that follows).

Reply:
that still leaves doubt though about when it does or doesn't, so authors and testers still are none the wiser